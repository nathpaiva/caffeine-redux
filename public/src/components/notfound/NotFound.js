import React from 'react';

const style = {
  title: {
    fontSize: '40px',
    textAlign: 'center',
    margin: '40px 0',
  },
  container: {
    maxWidth: '500px',
    width: '100%',
    display: 'flex',
    margin: 'auto',
    justifyContent: 'center',
    fontSize: '13px',
  },
  list: {
    margin: '0 5px'
  }
}

const NotFound = () => (
  <div>
    <h3 style={style.title}>Page not found :(</h3>
    <ul style={style.container}>
      <li style={style.list}><a href="/">Login</a></li>
    </ul>
  </div>
);

export default NotFound;
