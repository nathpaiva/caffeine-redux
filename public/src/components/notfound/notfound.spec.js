'use strict';

import React from 'react';
import renderer from 'react-test-renderer';

import NotFound from './NotFound';

describe('#Test Page NotFound', () => {
  it('#Should Page NotFound', () => {
    const tree = renderer.create(<NotFound />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
