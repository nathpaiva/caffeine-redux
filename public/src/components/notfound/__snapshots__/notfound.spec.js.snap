// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#Test Page NotFound #Should Page NotFound 1`] = `
<div>
  <h3
    style={
      Object {
        "fontSize": "40px",
        "margin": "40px 0",
        "textAlign": "center",
      }
    }
  >
    Page not found :(
  </h3>
  <ul
    style={
      Object {
        "display": "flex",
        "fontSize": "13px",
        "justifyContent": "center",
        "margin": "auto",
        "maxWidth": "500px",
        "width": "100%",
      }
    }
  >
    <li
      style={
        Object {
          "margin": "0 5px",
        }
      }
    >
      <a
        href="/"
      >
        Login
      </a>
    </li>
  </ul>
</div>
`;
